src/Database/PostgreSQL/PQTypes/Format.hs

```diff
@@ -1020,3 +1029,88 @@ instance
       , pqFormat @t49, pqFormat @t50
       ]
     pqVariables = 50
+
+instance
```

Comment:
No more instances please, types repeat in tests in these large tuples, just put Word* to replace some of the repeated occurrences.

Reply:
Got it!